src/index.jsx

```diff
@@ -2,9 +2,9 @@ import 'core-js/stable';
 import 'regenerator-runtime/runtime';
 
 import {
-  APP_INIT_ERROR, APP_READY, subscribe, initialize,
+  APP_INIT_ERROR, APP_READY, subscribe, initialize, mergeConfig,
 } from '@edx/frontend-platform';
-import { AppProvider, ErrorPage, PageRoute } from '@edx/frontend-platform/react';
+import { AppProvider, AuthenticatedPageRoute, ErrorPage } from '@edx/frontend-platform/react';
```

Comment:
Dumb question... is there a big significance between `PageRoute` and `AuthenticatedPageRoute`? Does `AuthenticatedPageRoute` mean the page is only accessible if you're logged in/not anonymous?

Reply:
Correct!

Reply:
We can always restrict it based on what returns from `courseHomeMetaData` api. That returns an `is_staff` boolean that we can use to restrict this page from the user.